monai/transforms/croppad/array.py

```diff
@@ -61,60 +70,69 @@
 class Pad(Transform):
```

Comment:
this class is not in the docs/; could you please help add?

Reply:
Added in the latest commit.
Thanks.